### PR TITLE
Added nav bar & script, cards images as example

### PR DESCRIPTION
**Pull request by aquamarineru**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+document.addEventListener("DOMContentLoaded", function(){
```
Comment (kaybee24) [inline script.js:1]: do we need the javascript? is it for fixed top navigation?
can't we use the 'fixed-top' from bootstrap? 
just wondering :) maybe i misunderstand.

File: index.html [hunk 2]
```diff
@@ -4,13 +4,112 @@
...
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+              </nav>
+        </div>
+    </header>
+    <section class="cards">
+        <div class="container">
+            <div class="row row-cols-1 row-cols-md-4 g-4">
```
Comment (kaybee24) [inline index.html:72]: didn't know about the row-cols - cool!

**Review by kaybee24 — COMMENTED** [review]
fine for me

**Review by fab-elmar — APPROVED** [review]

**Review by ArthiRagothaman — APPROVED** [review]
changes done

**Pull request merged**